New versions of bfd.h break build on Linux and Mac OS systems. Including
bfd.h requires constants to be defined, that are provided by autoconf.
Including autoconf.h into the files fixes the error.

https://sourceforge.net/p/avarice/patches/26/
--- src/jtag2prog.cc.orig
+++ src/jtag2prog.cc
@@ -38,6 +38,7 @@
 #include <math.h>
 
 #if ENABLE_TARGET_PROGRAMMING
+#  include "autoconf.h"
 #  include <bfd.h>
 #endif
 
--- src/jtagprog.cc.orig
+++ src/jtagprog.cc
@@ -37,6 +37,7 @@
 #include <math.h>
 
 #if ENABLE_TARGET_PROGRAMMING
+#  include "autoconf.h"
 #  include <bfd.h>
 #endif
 
